src/routes/DictionaryScreen.tsx

```diff
@@ -33,13 +34,23 @@ const Header = styled.View`
 `
 
 interface Props {
-  navigation: StackNavigationProp<RoutesParams, 'DictionaryOverview'>
+  navigation: StackNavigationProp<RoutesParams, 'Dictionary'>
 }
 
 const DictionaryScreen = ({ navigation }: Props): ReactElement => {
   const documents = useLoadAllDocuments()
   const [searchString, setSearchString] = useState<string>('')
-  const filteredDocuments = documents.data?.filter(item => item.word.includes(searchString))
+  const searchStringWithoutArticle = ARTICLES.map(article => article.value).includes(
+    searchString.split(' ')[0].toLowerCase()
+  )
+    ? searchString.substring(searchString.indexOf(' ') + 1)
+    : searchString
```

Comment:
This probably breaks for e.g. `Fön der` or just `der`. Tbh I just keep all of the query and not try to exclude articles since there is not really a need for this (users can either type the right articles or just omit the article in the search which should be the standard).

Reply:
When we want to search for a word with article, we would need to concat article + word of all documents (plus alternatives of documents), which is expensive / takes long. 
Just cutting the article is faster and therefore better.

Reply:
Does this work if just `die` is entered because e.g. searching for `Dietrich`?

Reply:
Yes it should work, as we only cut the "die", if the search string contains of more than one word and the first word is "die".

Tested, works as expected.